cmd/ledgerfsck/main.go

```diff
@@ -283,12 +298,13 @@ func (fsck *ledgerFsck) GetLatestChannelConfigBundle() error {
 func (fsck *ledgerFsck) Verify() {
 	blockchainInfo, err := fsck.ledger.GetBlockchainInfo()
 	if err != nil {
-		logger.Errorf("could not obtain blockchain information "+
+		logger.Debugf("could not obtain blockchain information "+
```

Comment:
Why is this Error log entry changed to Debug?

Reply:
That's for filtering log output from this command into 'FAIL' or 'PASS'. This command is mainly used from shell script and the script doesn't take care of the other additional messages.